docs/release-notes/delete-tensorboards.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Tensorboards: Fix an issue where tensorboard files were not deleted, even though their respective
+   experiments were via `det e delete <exp-id>`.
```

Comment:
:orphan:

**Bug Fixes**

-  TensorBoard: Fix an issue where TensorBoard files were not getting deleted even though the user deleted the experiment by using `det e delete <exp-id>`.